File: Faltantes o por revisar/Matematica/PossibleTimings.py
num=[2, 7, 2, 3, 3, 4, 2, 5, 1, 2]
def possibleTimings(n):
    print(num[(ord(n[0]) - ord('0'))] * num[ord(n[1]) - ord('0')]) 

n = "05"
possibleTimings(n)

File: Faltantes o por revisar/Matematica/PossibleTimings.cpp
#include "bits/stdc++.h" 
using namespace std;

int num[] = {2, 7, 2, 3, 3, 4, 2, 5, 1, 2};

void possibleTimings(string n) {
    cout << (num[(n.at(0) - '0')]
            * num[n.at(1) - '0']) << endl;
}

int main() {
    string n = "05";
    possibleTimings(n);
}
